lib/web_ui/test/initialization_test.dart

```diff
@@ -10,7 +11,8 @@ import 'package:ui/src/engine.dart' as engine;
 import 'package:ui/ui.dart' as ui;
 
 @JS('_flutter')
-external set loader(Object? loader);
+external set _loader(JSAny? loader);
+set loader(Object? l) => _loader = l?.toJSAnyShallow;
```

Comment:
Took me a while to figure out why this was a setter (it's a test 🤦)

Reply:
I may be misunderstanding something here, but isn't this redundant with the `js_util.jsify(...)` calls below?
```dart
set loader(Object? l) => _loader = l?.toJSAnyShallow;

// in tests:
loader = js_util.jsify(<String, Object>{
  'loader': <String, Object>{
    'didCreateEngineInitializer': allowInterop(() { print('not mocked'); }),
  },
});
```

Reply:
Ahh yea, good catch, it is redundant. I'm not actually trying to investigate usage patterns too deeply at this phase of the migration, mostly to derisk the conversion. That said, this is a good candidate for cleanup when we try to push JS types deeper into the Web engine. Thankfully this is a test, and `jsify` now has fast paths for types it doesn't need to convert.